### DISCUSSION TITLE
Daily 28/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#244] [General]
Ayer arreglé la cobertura de pruebas unitarias para que verifiquen el código real y no realice cobertura sobre las pruebas, así encontré varias partes del código que no estaban siendo testeadas por lo que agregué pruebas para cubrirlas. También renombre los controladores a servicios de manera que ahora el controlador principal es main.py y en la carpeta de controladores están los servicios que son llamados desde el main, voy a ver si leo cómo implementar los ruteos y los uso para simplificar algo más el código. También avancé con la implementación de la base de datos lo más que pude aunque no quiero avanzar mucho porque todavía me deben algunos queries, el resto de los queries creo que los voy a hacer yo durante el día así puedo terminar y probar contra una base de datos real. Ayer estuve en clase de SGC y de full-stack, hoy voy a estar también en la clase de full-stack. Me gustaría terminar con todo lo que es código para la reunión semanal.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo los queries de base de datos. También estuve estudiando Front y Sgc. No pude avanzar con la issue de Python. Participé en las clases sincrónicas de SGC y FullStack.
Hoy no cuento con mucho tiempo, así que terminaré de subir los queries pendientes y estudiare para la evaluación de Front. Voy a dejar la issue de Python para mañana.

**Comment by AhumadaCeleste** [comment]
Ayer estudie front, vi las clases de sgc y full. Estuve instalando y probando los plugin en wordpres para sumar utilidades a la pag. de Almoneda. Inicio con eso ahora para luego seguir con las issue asignadas, y usar el resto del fin de semana para estudiar front y rendir. Esta mañana antes de ir a trabajar estuve nuevamente tratando de resolver la dificultad de subir imágenes a wordpress y aun no lo logro. Espero poder también resolverlo hoy.
